src/exoscale/cel/parser.clj

```diff
@@ -54,3 +54,5 @@
        (handle-output config)))
   ([input]
    (parse-eval {} input)))
+
+(parse-eval "true == true && true == 1")
```

Comment:
```suggestion
```

Reply:
I am using this to show the type of whatever is parsed for now, will remove once it's done